pengwin-setup.d/cpp-vs-clion.sh

```diff
@@ -51,7 +51,7 @@ function main {
       done
     fi
 
-    bash "${SetupDir}"/jetbrains-support.sh --yes "$@"
+    bash "${SetupDir}"/jetbrains_support.sh --yes "$@"
```

Comment:
The standard in pengwin is to have dashes in the filenames, do please change it here and also the filename

Reply:
Done. 
I changed the file name and all the reference